sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/ExponentialHistogramIndexer.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.aggregator;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+class ExponentialHistogramIndexer {
+
+  private static final Map<Integer, ExponentialHistogramIndexer> cache = new ConcurrentHashMap<>();
+
+  /** Bit mask used to isolate exponent of IEEE 754 double precision number. */
+  private static final long EXPONENT_BIT_MASK = 0x7FF0000000000000L;
+
+  /** Bit mask used to isolate the significand of IEEE 754 double precision number. */
+  private static final long SIGNIFICAND_BIT_MASK = 0xFFFFFFFFFFFFFL;
+
+  /** Bias used in representing the exponent of IEEE 554 double precision number. */
```

Comment:
Is this a typo? Should be "IEEE 754"?

Reply:
Geez I can't seem to [get it right](https://github.com/open-telemetry/opentelemetry-java/pull/4700#discussion_r949674072) 😂